tests/test_log.py

```diff
@@ -1,8 +1,39 @@
-from vorta.log import LogTimedRotatingFileHandler
+import os
+import tempfile
+import time
+from logging.handlers import TimedRotatingFileHandler
 
+from vorta.log import log_namer
 
-def test_log_file_suffix():
-    """Ensure log files always have '.log' suffix."""
 
-    handler = LogTimedRotatingFileHandler('mylog.log', when='d', interval=1, backupCount=5)
-    assert handler.suffix.endswith('.log')
+def test_log_namer():
+    """Tests log_namer to ensure adding '.log' to end of filename did not break
+    any other functionality, such as creating and rotating/deleting backups"""
+
+    # Create a temporary directory for testing
+    with tempfile.TemporaryDirectory() as temp_dir:
+        log_dir = os.path.join(temp_dir, 'logs')
+        os.makedirs(log_dir)
+
+        # Create a handler and set its namer to the function we are testing
+        # 'when' and 'backupCount' set to speed up test execution time
+        handler = TimedRotatingFileHandler(os.path.join(log_dir, 'vorta.log'), when='s', interval=1, backupCount=3)
+        handler.namer = log_namer
+
+        # Test doRollover() which calls getFilesToDelete()
+        # Both of these functions rely on 'namer' working as intended
+        if handler.backupCount > 0:
+            for i in range(handler.backupCount + 2):  # Run extra times to ensure proper rotation of excess backups
+                handler.doRollover()
+                if i == 0:
+                    assert len(os.listdir(log_dir)) == 2  # Should keep initial backup + current log
+                else:
+                    if i < handler.backupCount:
+                        assert len(os.listdir(log_dir)) == i + 1  # Should keep i backups + current log
+                    else:
+                        assert len(os.listdir(log_dir)) == handler.backupCount + 1  # Keeps backupCount backups + log
+
+                for file in os.listdir(log_dir):
+                    assert file.endswith('.log')  # Checks that 'namer' is working as intended
```

Comment:
The current log is 'vorta.log', and the backups are 'vorta.log.[timestamp].log'

Reply:
> but removing this in namer messed with other functionality.

What functionality do you mean?

Reply:
The base filename is `vorta.log`, of which a timestamp gets appended on to. The `namer` function then appends `.log` to the end of this resulting in `vorta.log.[timestamp].log`

Unfortunately, if I implement functionality in the `namer` function to remove the superfluous `.log` , the `getFilesToDelete()` method in `TimedRotatingFileHandler` no longer properly identifies the files to delete, resulting in backups not rotating as intended. 

Additionally, if I set the base filename to simply 'vorta' instead of 'vorta.log', then the current log file will not get the '.log' suffix added since only rotated log files go through the `namer` function.